delegis:0.2.0

pklaschka opened the pull request:
<!--
Thanks for submitting a package! Please read and follow the submission guidelines detailed in the repository's README and check the boxes below. Please name your PR as `name:version` of the submitted package.

If you want to make a PR for something other than a package submission, just delete all this and make a plain PR.
-->

I am submitting
- [ ] a new package
- [x] an update for a package

<!--
Please add a brief description of your package below and explain why you think it is useful to others. If this is an update, please briefly say what changed.
-->

Description: 

#### Features
- Add `#metadata` fields for usage with `typst query`. You can now use `typst query file.typ "<field>" --field value --one` with `<field>` being one of the following to query metadata fields in the command line:
  - `<title>`
  - `<abbreviation>`
  - `<resolution>`
  - `<in-effect>`
- Add `#section[§ 1][ABC]` function to enable previously unsupported special chars (such as `*`) in section headings. Note that this was previously possible using `#unnumbered[§ 1\ ABC]`, but the new function adds a semantically better-fitting alternative to this fix.
- Improve heading style rules. This also fixes an incompatibility with `pandoc`, meaning it's now possible to use `pandoc` to convert delegis documents to HTML, etc.
- Set the footnote numbering to `[1]` to not collide with sentence numbers.
#### Bug Fixes
- Fix a typo in the `str-draft` variable name that lead to draft documents resulting in a syntax error.
- Fix hyphenation issues with the abbreviation on the title page (hyphenation between the parentheses and the abbreviation itself)

elegaanz commented on packages/preview/delegis/0.2.0/template/main.typ:
```diff
@@ -0,0 +1,64 @@
+#import "@preview/delegis:0.1.0": *
```
The import here is outdated.

elegaanz commented on packages/preview/delegis/0.2.0/README.md:
````diff
@@ -0,0 +1,131 @@
+### Importing the Package
+
+```typst
+#import "@preview/delegis:0.1.0": *
````
Same for this import.

elegaanz commented on packages/preview/delegis/0.2.0/template/main.typ:
```diff
@@ -0,0 +1,64 @@
+#show: it => delegis(
+  // Metadata
+  title: "Vereinsordnung zu ABCDEF",
+  abbrev: "ABCDEFVO",
```
This argument was renamed to `abbreviation` I think.

elegaanz commented on packages/preview/delegis/0.2.0/README.md:
```diff
@@ -0,0 +1,131 @@
+#show: it => delegis(
+  // Metadata
+  title: "Vereinsordnung zu ABCDEF", // title of the law/bylaw/...
+  abbrev: "ABCDEFVO", // abbreviation of the law/bylaw/...
```
It should be renamed here too.

pklaschka commented:
@elegaanz Thanks for your review! Those were stupid, avoidable issues (as the `abbreviation` one resulted from me fixing it after a review in #420 , then forgetting to also adjust it in the source repo, and now copying the wrong version over again). Sorry about that!

I have addressed your comments. Thanks and again: sorry about that.

elegaanz commented:
No problem, thanks for fixing that.
